src/Data/Nat/Induction.agda

```diff
@@ -83,6 +83,22 @@ module _ {ℓ} where
 <-wellFounded : WellFounded _<_
 <-wellFounded = Subrelation.wellFounded ≤⇒≤′ <′-wellFounded
 
+-- A version of `<-wellFounded` that cheats by skipping building
+-- the first billion proofs. Use this when you require the function
+-- needs to evaluate fast.
+--
+-- IMPORTANT: You have to be a little bit careful to always make the
+-- function be strict in some other argument than the accessibility proof,
+-- otherwise you will have neutral terms unfolding a billion times before
+-- getting stuck.
+<-wellFounded-fast : WellFounded _<_
+<-wellFounded-fast = <-wellFounded-skip {1000000000}
```

Comment:
Can we make the argument to `<-wellFounded-skip` explicit given that we
explicitly pass it & then explicitly match on it? :D

Reply:
Done!